pkg/kubelet/kubelet.go

```diff
@@ -108,6 +110,9 @@ func NewMainKubelet(
 		etcdClient:                     etcdClient,
 		kubeClient:                     kubeClient,
 		rootDirectory:                  rootDirectory,
+		tmpfsRootDirectory:             path.Join(rootDirectory, "tmpfs"), // TODO: pass in from cli
```

Comment:
Rereading this again, I found some ambiguity; did you mean that you didn't think tmpfsRootDirectory needed to be a field in the Kubelet, and that it should just be a function that joined to the root dir?